include/boost/asio/windows_sspi/stream.hpp

```diff
@@ -332,25 +328,22 @@ template <typename NextLayer> class stream : public stream_base {
       return 0;
     }
 
-    auto sent = boost::asio::write(m_next_layer,
-                                   boost::asio::buffer(message),
-                                   boost::asio::transfer_exactly(message.size()),
-                                   ec);
+    auto sent = net::write(m_next_layer, net::buffer(message), net::transfer_exactly(message.size()), ec);
 
     boost::ignore_unused(sent);
     BOOST_ASSERT(sent == message.size());
     if (ec) {
       return 0;
     }
 
-    return boost::asio::buffer_size(buffers);
+    return net::buffer_size(buffers);
   }
 
   template <typename ConstBufferSequence, typename CompletionToken>
   auto async_write_some(const ConstBufferSequence &buffer, CompletionToken &&token) ->
-      typename boost::asio::async_result<typename std::decay<CompletionToken>::type,
-                                         void(boost::system::error_code, std::size_t)>::return_type {
-    return boost::asio::async_compose<CompletionToken, void(boost::system::error_code, std::size_t)>(
+      typename net::async_result<typename std::decay<CompletionToken>::type,
+                                 void(boost::system::error_code, std::size_t)>::return_type {
+    return net::async_compose<CompletionToken, void(boost::system::error_code, std::size_t)>(
```

Comment:
`async_compose` is not part of Networking TS so this should just read `boost::asio::async_compose`

Reply:
I see. So the main reason why you prefer aliasing `boost::asio` to `net` is because of Networking TS?

Hadn't thought of that, but I guess that's similar to `namespace fs = boost::filesystem`.

I'll change that.

Reply:
Three reasons for `net`:
1. To abstract the difference between standalone Asio and Boost.Asio
2. To indicate to the reader that the function or type is a candidate for standardization
3. Because it is shorter (and thus easier to read)

It doesn't affect the correctness, but I place value on being informative. It can help reviewers and people who are interested in learning. It also helps me keep tabs on where the differences are between Asio and the upcoming networking standard. It isn't strictly necessary.

Reply:
I definitely agree with the third point, but just hadn't considered the two other points. It definitely makes sense.

I find it important that code is easy to read. It's hard to convince yourself that the code is correct if you have a hard time reading it in the first place, so I definitely find things like these important as well.